src/datasets.c

```diff
@@ -518,13 +517,16 @@ static int DatasetLoadString(Dataset *set)
         if (r == NULL) {
             line[strlen(line) - 1] = '\0';
             SCLogDebug("line: '%s'", line);
-
-            // coverity[alloc_strlen : FALSE]
```

Comment:
Why are you removing this comment for coverity ?

Reply:
This was accidental